OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -79,6 +79,7 @@ public interface IOrderTargeter
 		int OrderPriority { get; }
 		bool CanTarget(Actor self, Target target, List<Actor> othersAtTarget, TargetModifiers modifiers, ref string cursor);
 		bool IsQueued { get; }
+		bool OverrideSelection();
```

Comment:
To be honest, I don't have much experience with C# on a whole so a method was just what came to mind. I changed it to a getter if that helps since you're probably right that it won't be any more complex than this. As for the name, I think it's clear enough, or do you think that it should be denoted that it's used for left click orders?